packages/notebook/src/toc.ts

```diff
@@ -21,7 +21,11 @@ export enum RunningStatus {
   /**
    * Cell is idle
    */
-  Idle = -1,
+  Idle = -2,
+  /**
+   * Cell execution is unsuccessful
+   */
+  Error = -1,
```

Comment:
Could you set this to `-2` and leave `-1` on _Idle_ otherwise this is breaking the API (as it impacts also the DOM).

Reply:
If Idle is greater than Error and there are multiple cells under one heading, the heading will show as Idle instead of Error given how `getMaxIsRunning` is coded right now. Logically Idle should also be at the lowest priority. What API is it breaking? If we do have to work around that, we probably need to assign Error as -0.5.

Reply:
Ok let's go for `-0.5`

---

> What API is it breaking?

The enum number is directly set into the html data attribute. So strictly speaking if `Idle` value changes, the selector changes. I know this is being picky but I have seen consumer doing convoluted things.